src/main/java/com/redhat/devtools/lsp4ij/LanguageServersRegistry.java

```diff
@@ -84,6 +85,8 @@ public static LanguageServersRegistry getInstance() {
 
     private final Set<Language> customLanguageFindUsages = new HashSet<>();
 
+    private final SimpleModificationTracker modificationTracker = new SimpleModificationTracker();
```

Comment:
This is the first of two modification trackers that I've added, this one at the language server registry-level that's incremented with language server definitions are added and removed.